pkg/controllers/clusters/clusterstatus/clusterstatus_controller.go

```diff
@@ -222,20 +222,28 @@ func (c *Controller) discoverClusterCIDR() (string, error) {
 
 // get node capacity and allocatable resource
 func getNodeResource(nodes []*corev1.Node) (Capacity, Allocatable corev1.ResourceList) {
-	var capacityCpu, capacityMem, allocatableCpu, allocatableMem resource.Quantity
+	var capacityCpu, capacityMem, capacityGpu, allocatableCpu, allocatableMem, allocatableGpu resource.Quantity
 	Capacity, Allocatable = make(map[corev1.ResourceName]resource.Quantity), make(map[corev1.ResourceName]resource.Quantity)
 
 	for _, node := range nodes {
 		capacityCpu.Add(*node.Status.Capacity.Cpu())
 		capacityMem.Add(*node.Status.Capacity.Memory())
 		allocatableCpu.Add(*node.Status.Allocatable.Cpu())
 		allocatableMem.Add(*node.Status.Allocatable.Memory())
+		if _, exists := node.Status.Capacity[known.NVIDIAGPUResourceName]; exists {
+			capacityGpu.Add(node.Status.Capacity[corev1.ResourceName(known.NVIDIAGPUResourceName)])
+			allocatableGpu.Add(node.Status.Allocatable[corev1.ResourceName(known.NVIDIAGPUResourceName)])
+		}
 	}
 
 	Capacity[corev1.ResourceCPU] = capacityCpu
 	Capacity[corev1.ResourceMemory] = capacityMem
 	Allocatable[corev1.ResourceCPU] = allocatableCpu
 	Allocatable[corev1.ResourceMemory] = allocatableMem
+	if !allocatableGpu.IsZero() {
```

Comment:
should be `capacityGpu`, instead of `allocatableGpu` ?

Reply:
I know `capacityGpu` looks more consistent. However i found a case, when nvidia-gpu-plugin removed, the value of `allocatableGpu` is 0, however `capacityGpu` is 2, It last about 5 minutes until `allocatableGpu` and `capacityGpu`  all become to 0.  I don't know if that is a nvidia-gpu-plugin bug or designed to be.  So I used `allocatableGpu` rather than `capacityGpu `.